tests/phpunit/tests/base-notification.php

```diff
@@ -2,7 +2,7 @@
 
 class Test_WPNotify_BaseNotification extends WPNotify_TestCase {
 
-	const JSON_SERIALIZED = '{"WPNotify_RecipientCollection":[],"WPNotify_BaseMessage":"Message","WPNotify_BaseSender":{"name":"Name 1"}}';
+	const JSON_SERIALIZED = '{"WPNotify_RecipientCollection":[],"WPNotify_BaseMessage":"Message","WPNotify_BaseSender":{"name":"Name 1","image":null}}';
```

Comment:
Same comment than for the `Test_WPNotify_BaseImage::data_provider_images()`: maybe we don't want the empty `"image"` field to be printed in our JSON string. :thinking:

Reply:
fixed